Mathlib/CategoryTheory/Adjunction/Mates.lean

```diff
@@ -39,12 +41,11 @@ This also relates to Grothendieck's yoga of six operations, though this is not s
 mathlib: https://ncatlab.org/nlab/show/six+operations.
 -/
 
-
-universe v₁ v₂ v₃ v₄ u₁ u₂ u₃ u₄
+universe v₁ v₂ v₃ v₄ v₅ v₆ v₇ v₈ v₉ u₁ u₂ u₃ u₄ u₅ u₆ u₇ u₈ u₉
```

Comment:
That's a lot of new universes! Can we have some of them declared in the section they're used in (I think it's only in composition?)

Reply:
I wasn't sure what the best convention is. All the universes at the top or just the necessary universes at the start of each section?